sci-calculators/bc-gh/bc-gh-1.2.6.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Implementation of POSIX bc with GNU extensions"
+HOMEPAGE="https://github.com/gavinhoward/bc"
+SRC_URI="https://github.com/gavinhoward/bc/releases/download/${PV}/bc-${PV}.tar.xz"
+
+LICENSE="BSD-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86"
```

Comment:
You can't add KEYWORDS for systems you haven't tested this on, and even with some more exotic arches, you need to belong in the arch team before you can add support for them.

Reply:
Eh, sorry about that. I just copied and pasted. My bad.

Note that I kept the `KEYWORDS` that I did because my bc builds and runs on those architectures. See http://pkg.musl.cc/bc-1.2.6/ for more information.